apps/02RockPaperScissors.js

```diff
@@ -6,47 +6,35 @@ prompt.start();
 
 
 function rockPaperScissors(hand1, hand2) {
-if (hand1 === hand2)  {
-  return "It\'s a tie!"
-}
+    hand1 = hand1.toLowerCase();
+    hand2 = hand2.toLowerCase();
 
-else if (
-  (hand1 === 'rock' && hand2 === 'scissors') ||
-  (hand1 === 'scissors' && hand2 === 'paper') ||
-  (hand1 === 'paper' && hand2 === 'rock')
-) {
-  return "Hand one wins!";
-}
+//var valid = ['rock', 'paper', 'scissors'];
 
-else {
-  return "Hand two wins!";
-}
+if (hand1 === 'rock' || hand1 === 'scissors' || hand1 === 'paper'
+&&  hand2 === 'rock' || hand2 === 'scissors' || hand2 === 'paper')  {
+
+      if (hand1 === hand2)  {
+        return "It\'s a tie!"
+      }
+
+      else if (
+        (hand1 === 'rock' && hand2 === 'scissors') ||
+        (hand1 === 'scissors' && hand2 === 'paper') ||
+        (hand1 === 'paper' && hand2 === 'rock')
+      ) {
+        return "Hand one wins!";
+      }
 
-    // if (hand1 === hand2) {
-    //   return "It's a tie!";
-    // }
-    //
-    // else if (hand1 === 'rock')  {
-    //   if (hand2 === 'scissors') {
-    //     return "Hand one wins!";
-    //   }
-    // }
-    //
-    // else if (hand1 === 'paper') {
-    //   if (hand2 === 'rock') {
-    //       return "Hand one wins!";
-    //   }
-    // }
-    //
-    // else if (hand1 === 'scissors')
-    //   if (hand2 === 'paper')  {
-    //     return "Hand one wins!";
-    //   }
-    // else {
-    //     return "Hand two wins!";
-    //   }
-    // }
+      else {
+        return "Hand two wins!";
+      }
 
+
+      }
+else {
+  return "Please enter a valid input."
+}
 };
 
```

Comment:
Yep! Its always good to assume the user is going to just slam their fist on the keyboard and send whatever they get. It helps prevent your program from breaking and is a form of "error handling."